tests/functional/spec/features/submit_page/submit_page_collapsible_summary.spec.js

```diff
@@ -22,9 +22,9 @@ describe("Collapsible Summary", () => {
       $(HouseholdCountSectionSummaryPage.submit()).click();
     });
 
-    it("When I am on the Final Summary, Then the group titles should be displayed", () => {
-      expect($(QuestionnaireSummaryPage.collapsibleSummary()).getText()).to.contain("Property Details");
-      expect($(QuestionnaireSummaryPage.collapsibleSummary()).getText()).to.contain("House Details");
+    it("When I am on the submit page, Then the group titles should be displayed", () => {
+      expect($(SubmitPage.collapsibleSummary()).getText()).to.contain("Property Details");
+      expect($(SubmitPage.collapsibleSummary()).getText()).to.contain("House Details");
```

Comment:
An existing issue with the test, but it's not clear how this is testing the "collapsible" bit.

Reply:
I guess it was because `collapsibleSummary` didn't fail. But ya it's not great. While I am here, I made a slight extension to these tests to be a bit more meaningful. I haven't done much as I didn't want to go on a tangent.